TourMate/TourMate/Frontend/Plan/Models/Plan.swift

```diff
@@ -7,63 +7,18 @@
 
 import Foundation
 
-struct Plan: CustomStringConvertible {
-    let id: String
-    let tripId: String
-    var name: String
-    var startDateTime: DateTime
-    var endDateTime: DateTime
-    var startLocation: Location?
-    var endLocation: Location?
-    var imageUrl: String
-    var status: PlanStatus
-    let creationDate: Date
-    let modificationDate: Date
-    var additionalInfo: String
-    var ownerUserId: String
-
-    init(id: String, tripId: String, name: String,
-         startDateTime: DateTime, endDateTime: DateTime,
-         startLocation: Location? = nil, endLocation: Location? = nil,
-         imageUrl: String = "", status: PlanStatus, additionalInfo: String = "",
-         ownerUserId: String) {
-        self.id = id
-        self.tripId = tripId
-        self.name = name
-        self.startDateTime = startDateTime
-        self.endDateTime = endDateTime
-        self.startLocation = startLocation
-        self.endLocation = endLocation
-        self.imageUrl = imageUrl
-        self.status = status
-        self.creationDate = Date()
-        self.modificationDate = Date()
-        self.additionalInfo = additionalInfo
-        self.ownerUserId = ownerUserId
-    }
-
-    init(id: String, tripId: String, name: String,
-         startDateTime: DateTime, endDateTime: DateTime,
-         startLocation: Location? = nil, endLocation: Location? = nil,
-         imageUrl: String = "", status: PlanStatus,
-         creationDate: Date, modificationDate: Date,
-         additionalInfo: String = "",
-         ownerUserId: String) {
-        self.id = id
-        self.tripId = tripId
-        self.name = name
-        self.startDateTime = startDateTime
-        self.endDateTime = endDateTime
-        self.startLocation = startLocation
-        self.endLocation = endLocation
-        self.imageUrl = imageUrl
-        self.status = status
-        self.creationDate = creationDate
-        self.modificationDate = modificationDate
-        self.additionalInfo = additionalInfo
-        self.ownerUserId = ownerUserId
-    }
-
+protocol Plan: CustomStringConvertible {
```

Comment:
Why is this a protocol now? I think if we leave it as plan we have less duplication